GitSlicePullPlatform/src/main.ts

```diff
@@ -0,0 +1,131 @@
+import * as core from '@actions/core'
+import {context} from '@actions/github'
+import axios from 'axios'
+import {promises as fs} from 'fs'
+
+process.on('unhandledRejection', handleError)
+main().catch(handleError)
+
+function conditionalBoolean(strBoolean: string | undefined) {
+  return strBoolean === 'true'
+    ? true
+    : strBoolean == 'false'
+    ? false
+    : undefined
+}
+
+async function main(): Promise<void> {
+  // Read environment values
+  const slicedRepoToken = core.getInput('slice_git_token', {
+    required: false
+  })
+  const slicedRepoUsername = core.getInput('slice_git_username', {
+    required: false
+  })
+  const slicedRepoBranch = core.getInput('slice_default_branch', {
+    required: true
+  })
+  const slicedRepoOwner = context.repo.owner
+  const slicedRepoName = context.repo.repo
+
+  const upstreamRepoToken = core.getInput('upstream_git_token', {
+    required: false
+  })
+  const upstreamRepoUsername = core.getInput('upstream_git_username', {
+    required: false
+  })
+  const upstreamRepoEmail = core.getInput('upstream_git_email', {
+    required: true
+  })
+
+  const is_open_source = core.getInput('is_open_source', {
+    required: false
+  })
+  const gitSliceFile = await fs.readFile('./git-slice.json')
+
+  // Prepare request
+  const endpoint = 'https://git-service.gitstart.dev/graphql'
```

Comment:
can this be conditional ... based on an env var perhaps? or perhaps we can go straight to .com?

Reply:
I'll change it to .com after I release to prod